doc/Learn/Markup/Overview.md

```diff
@@ -175,7 +175,7 @@ public sealed partial class MainPage : Page
 
 ## Conclusion
 
-This overview has provided a brief introduction to C# Markup with an introduction to how to create controls and set properties, as well as how to use data binding and resources. Check out the full [Counter sample](TBD) to see how to use C# Markup to create a complete application. Check out the [C# Markup reference](TBD) for more information on available extension methods.
+This overview has provided a brief introduction to C# Markup with an introduction to how to create controls and set properties, as well as how to use data binding and resources. Check out the full [Counter tutorial](https://aka.platform.uno/counter-tutorial) to see how to use C# Markup to create a complete application. Check out the [C# Markup reference](TBD) for more information on available extension methods.
```

Comment:
@dansiegel, @nickrandolph 
What would be the proper link for [C# Markup reference](TBD) here please